libweb3jsonrpc/WebThreeStubServerBase.cpp

```diff
@@ -284,6 +284,11 @@ bool WebThreeStubServerBase::net_listening()
 	return network()->isNetworkStarted();
 }
 
+string WebThreeStubServerBase::eth_version()
```

Comment:
rename to `eth_protocolVersion` (version is a bit too vague, given the plethora of version ids we have)